### DISCUSSION TITLE
Is there a way to stop auto complete when typing triple quotes?

### DISCUSSION THREAD

**Discussion by huntfx** [microsoft/pylance-release#1984] [Q&A] [answered]
Sometimes I'll select a block of code to comment it out, but most of the time due to indents I just manually write the triple quotes before and after. 

Recently, it's started auto completing 3 quotes to 6, and I end up having to go back to manually delete the 6 extra quotes every time, which is getting quite annoying.

Is there a way I could disable that feature? I'm presuming it's pylance as it goes away if I disable the extension.

Example of how most of my commented out blocks initially end up:

```python
''''''
# code I want to comment out
''''''
```

**Comment by bschnurr** [comment, marked as answer]
search for 'autoClosingQuotes' in vscode settings

 "editor.autoClosingQuotes": "never"

**Reply by jakebailey** [reply]
That setting will work to disable the quoting, but will disable all auto-quoting. @huntfx can you create a full issue for a request to add something more fine?

**Reply by huntfx** [reply]
Thanks, I'll change that setting as it seems preferable to the behaviour I mentioned earlier, will make the request though as overall I'd prefer to keep it enabled for single quotes.
